ts/node/database_utility.ts

```diff
@@ -0,0 +1,183 @@
+import { difference, omit, pick } from 'lodash';
+import { ConversationAttributes } from '../models/conversationAttributes';
+
+export function objectToJSON(data: Record<any, any>) {
+  const str = JSON.stringify(data);
+  return str;
+}
+export function jsonToObject(json: string): Record<string, any> {
+  return JSON.parse(json);
+}
+
+function jsonToArray(json: string): Array<string> {
+  try {
+    return JSON.parse(json);
+  } catch (e) {
+    console.warn('jsontoarray failed:', e.message);
+    return [];
+  }
+}
+
+export function arrayStrToJson(arr: Array<string>): string {
+  return JSON.stringify(arr);
+}
+
+export function toSqliteBoolean(val: boolean): number {
```

Comment:
well line `86-93` actually need the opposite right?
They take a value and return a boolean.

so that would be a 

```ts
fromSqliteBoolean(val: any): boolean {
    return Boolean(val);
}
```

Is this what you think?

Reply:
Ah yes my mistake! In that case leave it is as. We don't need `romSqliteBoolean` unless you want it to be more explicit with what the code is doing.